src/lib/forms/PipelineForm.svelte

```diff
@@ -18,8 +18,14 @@
     atf4_type: "",
     accuracy: "",
     is_alternative: false,
+    is_coverage_polygon: false,
   };
 
+  const shouldDisplayCoveragePolygonQuestion: boolean =
+    $gjScheme.features.filter(
```

Comment:
Also only if the current feature `geometry.type == "Polygon"` -- no point for lines/points